ngDraggable.js

```diff
@@ -413,8 +414,8 @@ angular.module("ngDraggable", [])
                     var onDragMove = function(evt, obj) {
                         if(_allowClone) {
 
-                            _tx = obj.tx + _dragOffset.left;
-                            _ty = obj.ty + _dragOffset.top;
+                            _tx = obj.x - document.body.scrollLeft - (obj.element[0].offsetWidth/2);// + _dragOffset.left;
```

Comment:
scrollLeft and scrollTop does not work in FF
you should use window.pageXOffset and window.pageYOffset instead (works in all good browsers and IE9+)
https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY